site/docs/relations/logical_relations.md

```diff
@@ -345,13 +345,13 @@ The aggregate operation groups input data on one or more sets of grouping keys,
 | Inputs               | 1                                                            |
 | Outputs              | 1                                                            |
 | Property Maintenance | Maintains distribution if all distribution fields are contained in every grouping set. No orderedness guaranteed. |
-| Direct Output Order  | The list of distinct columns from each grouping set (ordered by their first appearance) followed by the list of measures in declaration order, followed by an `i32` describing the associated particular grouping set the value is derived from (if applicable). |
+| Direct Output Order  | The list of grouping expressions in declaration order followed by the list of measures in declaration order, followed by an `i32` describing the associated particular grouping set the value is derived from (if applicable). |
```

Comment:
An added benefit of this is that it makes it much easier to compute the output order. Declaration order is very clear ✨